.travis.yml

```diff
@@ -1,17 +1,14 @@
 language: ruby
+sudo: false
+cache: bundler
+bundler_args: '--without production development'
 
 rvm:
   - 2.2.2
 
-env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
```

Comment:
What I try with this is to have the CI configured in the same place as the other. So that we avoid introducing any differences between them.